### PR TITLE
Correct typos in the codecs module documentation

### PR DESCRIPTION
**Pull request by geryogam**
This PR will correct typos and punctuation in the [`codecs` module documentation](https://docs.python.org/3/library/codecs.html).

### CODE REVIEW

**Comment by maggyero** [comment]
@willingc If you have some time, could you also review this one?

**Comment by willingc** [comment]
Hi @maggyero. Would you please resubmit this PR without the whitespace changes? It's acceptable to have one or two spaces following a period. It helps a reviewer save time if only the typo corrections or text changes are submitted in a PR. Thanks!

**Review by willingc — CHANGES_REQUESTED** [review]
Please resubmit without whitespace changes. Thanks @maggyero.

**Comment by maggyero** [comment]
Hi @willingc. Thank you for looking at this. Single and double spaces were mixed in this file, that is why I normalized them in my last commit in the first place. But you are right, it is harder to review when there are other corrections involved in the PR, so I reverted them.

I have made the requested changes; please review again.

**Comment by maggyero** [comment]
@willingc

File: Doc/library/codecs.rst [hunk 1]
```diff
@@ -841,8 +841,8 @@ The design is such that one can use the factory functions returned by the
    code calling :meth:`read` and :meth:`write`, while *Reader* and *Writer*
    work on the backend — the data in *stream*.
 
-   You can use these objects to do transparent transcodings from e.g. Latin-1
-   to UTF-8 and back.
+   You can use these objects to do transparent transcodings, e.g., from Latin-1
+	to UTF-8 and back.
```
Comment (willingc) [inline Doc/library/codecs.rst]: This is failing the lint check for tabs.

**Comment by willingc** [comment]
@maggyero Travis is reporting errors in linting
```
[1] library/codecs.rst:845: OMG TABS!!!1
1 problem with severity 1 found.
```
Please review and update. Once Travis is green for all checks, please ping me again.

File: Doc/library/codecs.rst (hunk 1 above)
Comment (geryogam) [reply Doc/library/codecs.rst]: Oops, thanks!

**Comment by maggyero** [comment]
@willingc Thanks, fixed.

**Comment by maggyero** [comment]
@willingc Thanks, fixed. I have made the requested changes; please review again.

**Comment by maggyero** [comment]
@willingc

**Review by willingc — APPROVED** [review]
Thanks @maggyero.

**Comment by animalize** [comment]
Could you backport this PR to the 3.7 and 3.8 branches?
I have another PR https://github.com/python/cpython/pull/15732 base on this PR.

**Comment by maggyero** [comment]
@willingc

**Comment by willingc** [comment]
About to board a flight. If anyone wishes to cherry-pick this, that would be great. Otherwise, it will need to wait until tomorrow.

**Comment by animalize** [comment]
No hurry, take your time.
It doesn't matter even next week.

**Comment by maggyero** [comment]
Hello @willingc. Are you still available to finalize the backport? Or do you prefer that we ask someone else?

**Comment by maggyero** [comment]
Hi @willingc. May I request your time to finalize the backport to Python 3.7 and to add the backport label for Python 3.8?

**Comment by maggyero** [comment]
Thanks @JulienPalard for backporting this.

**Pull request merged**